geoh5py/objects/geo_image.py

```diff
@@ -379,3 +379,54 @@ def to_grid2d(
                 }
             )
         return grid
+
+    @property
+    def image_georeferenced(self) -> Image.Image | None:
+        """
+        Get the image as a georeferenced :obj:`PIL.Image` object.
+        """
+        if self.tag is not None and self.image is not None:
+            image = self.image
+
+            # modify the exif
+            for id_ in self.tag:
+                image.getexif()[id_] = self.tag[id_]
+
+            return image
+
+        return None
+
+    def export_as_georeferenced_tif(
+        self, name: str, path: str = ""
+    ):  # todo: export as a normal tiff if no tag?
```

Comment:
Can we generalize this and just call `export`? We could check if the outgoing name is `tif`, then if `tag` exists we try to save as geotif, otherwise plain image with PIL?